tracks/create-reducer.es6

```diff
@@ -68,6 +68,7 @@ export default function createReducer({END, JUMP, LOAD, PAUSE, PLAY, STOP, TICK}
 
         nextTrack = {
           ...state[id],
+          didEnd: true,
```

Comment:
If that is the intended purpose, then we should remove it. I thought it meant that "the track has ended because it has been stopped" rather than the track has reached the end". TBH it was something I had in before we added the seek method call to reset the track. I can tidy it up in the morning.